templates: Update CMS Guides for Education and Research

RaoOfPhysics opened the pull request:
If you want to see what these look like, I've taken screenshots of the full pages. Feedback welcome. Let me know if I should make any changes and push to this branch.

**Education**:

![cms-guides-education](https://user-images.githubusercontent.com/7623019/33078360-bc426f20-ced2-11e7-9c13-5848bf48cabf.png)

**Research**:

![cms-guides-research](https://user-images.githubusercontent.com/7623019/33078379-c6707dca-ced2-11e7-80a7-697c900c2c7c.png)

RaoOfPhysics commented:
@tiborsimko, please let me know if hardcoding `<center>–––</center> is poor form. I can remove it if there's a better suggestion for how to break up the text within each "section".

RaoOfPhysics commented:
Also: All the "quote" in the Research page are enclosed within `<blockquote>`s. So, we can give them a different style if we want.

sefeg commented on cernopendata/modules/fixtures/data/articles/cms-guide-for-education/education.md:
```diff
@@ -1,73 +1,67 @@
 
+#### Dive deeper into the data
 
+If you want a more detailed understanding of particle physics or an introduction aimed at the university level, take a look at some of the other resources in [CMS Learning resources ](http://opendata.cern.ch/collection/CMS-Learning-Resources).
```
@tiborsimko Are collection pages like _http://opendata.cern.ch/collection/CMS-Learning-Resources_ ported to _/search?q=CMS-Learning-Resources_ ?

RaoOfPhysics replied:
@tiborsimko: Please let me know if I should change this before the PR is merged. Thanks!

sefeg commented:
Looks very good to me. I leave a review decision on this to @katilp

katilp commented:
Looks very good to me!

katilp commented:
Maybe replace the learning resource collection link with
http://opendata-qa.web.cern.ch/search?page=1&size=20&q=&keywords=education

RaoOfPhysics commented:
Thanks for the feedback, @sefeg and @katilp. Now fixed in https://github.com/cernopendata/opendata.cern.ch/pull/1848/commits/701db9cb8a536398615676202b7d97c1424cb929

@ioannistsanaktsidis: I think this is good to go. I've rebased with the latest `master`.
